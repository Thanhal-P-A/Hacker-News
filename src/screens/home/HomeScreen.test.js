import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import {Provider} from 'react-redux';
import {store} from '../../../App';

import Home from '.';

it('Home screen renders correctly', () => {
  const HomeScreen = renderer
    .create(
      <Provider store={store}>
        <Home />
      </Provider>,
    )
    .toJSON();
  expect(HomeScreen).toMatchSnapshot();
});