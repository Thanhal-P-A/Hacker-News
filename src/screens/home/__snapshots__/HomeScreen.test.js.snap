// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home screen renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "backgroundColor": "#ED642E",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "flex": 1,
        "justifyContent": "center",
        "padding": 10,
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#000000",
          }
        }
      >
        Please check your internet connection and retry
      </Text>
    </View>
  </View>
</RCTSafeAreaView>
`;
